metadata.go

```diff
@@ -80,12 +82,36 @@ var debianLicenseText = map[string]string{
 `,
 }
 
+var githubRegexp = regexp.MustCompile(`github\.com/([\w._-]+/[\w._-]+)`)
```

Comment:
How about `github\.com/([^/]+/[^/>" ]+)`? To elaborate on why I’m being so difficult about this regexp: in general, I’d like this program to be permissive by default (using a blacklist of forbidden characters), instead of restrictive by default (using a whitelist of allowed characters). That seems more robust to me.

If my latest suggestion doesn’t work either, we’ll stick with yours and I’ll implement the HTML parsing afterwards :).